### DISCUSSION TITLE
Feb 16

### DISCUSSION THREAD

**Discussion by miniposi** [GymHubCommunity/GymHub-FE#170] [DailyMeeting]
### ⚽️ 회의 내용

* 논의
  백: 운동 부위 백에서 관리해주시나요? => 프론트에서
  디자인: 검색 기능/디자인을 하나로 통일하면 좋을 거 같습니다.
  lighthouse 잘못 만들었다....

### 🏀 오늘 할 일

슬희: persigned url -> s3 -> post 요청
수: 팔로워/팔로잉 숫자 뜨게 연동, 메인화면 가운데, 운동기록 디자인 바뀐 거 적용
수영: persigned url -> s3 -> post 요청 -> 해시태그
윤혁: 게시물 상세 화면 마무리 (input..) 

### 🎱 TMI

슬희: 스타필드에서 비긴어게인 방청 신청 😵‍💫
수: 롤 연패 끊음.. 슬프다 😭
수영: 위워크 가는 중 🏃
윤혁: 배신 ㄷ ㄷ | 보편의 단어 📚
정우: 부지런치..🥪

**Comment by hanseulhee** [comment]
보통의 존재 메모 !!
